modules/kimberliteBidAdapter.js

```diff
@@ -0,0 +1,72 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js'
+
+const VERSION = '1.0.0';
+
+const BIDDER_CODE = 'kimberlite';
+const METHOD = 'POST';
+const ENDPOINT_URL = 'https://kimberlite.io/rtb/bid/pbjs';
+
+const converter = ortbConverter({
+  context: {
+    mediaType: BANNER,
+    netRevenue: true,
+    ttl: 300
+  },
+
+  request(buildRequest, imps, bidderRequest, context) {
+    const bidRequest = buildRequest(imps, bidderRequest, context);
+    bidRequest.site ||= {};
+    bidRequest.site.publisher ||= {
+      domain: bidderRequest.refererInfo.domain
+    };
+    bidRequest.site.page ||= bidderRequest.refererInfo.page;
+    bidRequest.at = 1;
+    bidRequest.ext = {
+      prebid: {
+        ver: '$prebid.version$',
+        adapterVer: `${VERSION}`
+      }
+    };
```

Comment:
We want to overwrite the `ext` object to just be this each time? Is that intended?

Reply:
You're right, we should extend it, yes. Thank you!